tests/test_source_unit.py

```diff
@@ -1,5 +1,7 @@
 from slither import Slither
 
+# NB: read tests/source_unit/README.md before using this test
```

Comment:
Could we detect if this was not done in the test itself, and raise a warning in pytest?

Reply:
I added a skipif, it will skip the test if deps are not met + log a skip reason

<img width="832" alt="Screenshot 2023-01-06 at 15 35 18" src="https://user-images.githubusercontent.com/2642849/211076562-52f53fd6-6887-4c78-9a89-a1154b2fd339.png">